src/dotnet-gcmon/dotnet-gcmon.csproj

```diff
@@ -14,7 +14,7 @@
 
   <PropertyGroup>
     <PackageId>dotnet-gcmon</PackageId>
-    <PackageVersion>0.2.0</PackageVersion>
+    <PackageVersion>0.3.0</PackageVersion>
```

Comment:
Should be updated to 0.4.0 since another PR with 0.3.0 has been merged

Reply:
do we need to change this if I publish a new version with this change and the other change?

Reply:
> do we need to change this if I publish a new version with this change and the other change?

I thought you already published the version with the other PR.
No problem to release to nuget when all PRs are merged so you avoid bumping the version too often